pkgs/development/mobile/cocoapods/default.nix

```diff
@@ -9,6 +9,13 @@ bundlerApp rec {
   gemset = if beta then ./gemset-beta.nix else ./gemset.nix;
   exes = [ "pod" ];
 
+  passthru.updateScript = writeScript "update-cocoapods" ''
+    #!${runtimeShell}
```

Comment:
It's not obvious to me what the difference between `runtimeShell` and `stdenv.shell` is, though they appear to resolve to the same package on my system.